### PR TITLE
Update docker env var during bootstrap

### PR DESCRIPTION
**Pull request by chrisjlebron**
#### Board:
* N/A
---
#### Description:

When using Docker, the setup script was failing / not working as expected. The `running_with_docker?` method checks the env var `DOCKER_ENABLED=true`, but the default is `false`. This adds a rewrite of that value if bootstrapping with Docker. I also added a utility method for the bootstrap script since this uses `sed` again, necessitating the OS check.

---
#### Notes:
* I'm new here, so I'm not sure if we want to load dotenv in all the bin scripts that reference `running_with_docker?`, but it seems like it might be worthwhile, if anyone runs those scripts in isolation.
---
#### Tasks:
N/A

---
#### Risk:
N/A

---
#### Preview:
N/A

### CODE REVIEW

**Comment by santib** [comment]
>I'm new here, so I'm not sure if we want to load dotenv in all the bin scripts that reference running_with_docker?, but it seems like it might be worthwhile, if anyone runs those scripts in isolation.

@chrisjlebron good catch, maybe we should `require dotenv/load` in `bin/helpers.rb` ? so we don't need to require it in N different places

File: bootstrap.sh [hunk 1]
```diff
@@ -58,16 +58,24 @@ fi
...
 cp .env.sample .env
 
-# update project name in database.yml
 # sed command implementation is different for GNU and macOS
-if [[ $OSTYPE == 'darwin'* ]]; then
-  sed -i '' "s/rails_api_base/${project_name}/g" config/database.yml
-else
-  sed -i "s/rails_api_base/${project_name}/g" config/database.yml
-fi
+sed_i() {
```
Comment (guillermoap) [inline bootstrap.sh:62]: Very nice

**Review by guillermoap — APPROVED** [review]

**Review by JulianPasquale — APPROVED** [review]
LGTM

**Comment by chrisjlebron** [comment]
> @chrisjlebron good catch, maybe we should `require dotenv/load` in `bin/helpers.rb` ? so we don't need to require it in N different places

Moved that, thanks for the suggestion

**Comment by santib** [comment]
Thanks @chrisjlebron 🙌

**Pull request merged**